UI/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<meta charset="UTF-8"/>
+		<title>Моя перва HTML страница</title>
+		<link href = "style.css" rel = "stylesheet" type = "text/css"/>
+	</head>
+	<body>
+		<div class = "Wallpaper"></div>
+		
+		
+		<div class = "Header"></div>
+		<div class = "HeaderIcon">HUKUMKA</div>
+		<div class = "HeaderUserName">username</div>
+		<div class = "HeaderRectangle"></div>
+		<a href = "https://vk.com/justakel" class = "HeaderLogin">sign out</a>
+		
+		
+		<div class = "Footer"></div>
+		<div class = "FooterIcon">HUKUMKA</div>
+		<div class = "FooterData">01.03.2021</div>
+		<div class = "FooterName">Hukumka Inc.</div>
+		<div class = "FooterInfo">2 course, 8 group</div>
+		<div class = "FooterEmail">nikitakondibor@gmail.com</div>
+		
+		
+		<div class = "Filter"></div>
+		<div class = "FilterName">filter by</div>
+		<div class = "FilterSaleRectangle"></div>
```

Comment:
For elements which have common functionality you can use common classes, so here you don't need to have separate "FilterSaleRectangle" and "FilterDataRectangle", you can use just "filter-rectangle". Also you can combine some classes if you write it as a list: "filter-sale filter-rectangle". 
Please note that all classes the most common way to write HTML classes is using 'kebab-case' (dashes as delimiters). CSS is case-insensitive language, so it won't understand the difference between 'filtername' and 'FilterName'.